annotations/split_manifest_grants.py

```diff
@@ -0,0 +1,63 @@
+"""Split Manifests CSV
+This script will split a manifest csv by grant number and output
+results into individual CSVs.
+author: verena.chung
+author: brynn.zalmanek
+"""
+
+import os
+import argparse
+import pandas as pd
+
+
+def get_args():
+    """Set up command-line interface and get arguments."""
+    parser = argparse.ArgumentParser()
+    parser.add_argument("manifest",
+                        type=str,
+                        help="path of manifest to be split")
+    parser.add_argument("manifest_type",
+                        type=str,
+                        choices=["publication", "dataset", "tool", "project"],
+                        help="type of manifest to split, e.g. publicaiton")
+    parser.add_argument("folder",
+                        type=str,
+                        help="folder path to save split manifests in")
+    return parser.parse_args()
+
+
+def get_df(manifest_csv):
+    """Convert manifest to data frame."""
+    df = pd.read_csv(manifest_csv)
```

Comment:
This function is a bit too simple.  It might be better to keep this line of code in `main()` instead, since functions are more meant for situations where:

* calling the function would look simpler than the code it wraps
* the line(s) of code will likely be used multiple times
* the line(s) of code are likely to change (if so, then using a function will help in only editing in one place rather than everywhere the line(s) of code are used)

Reply:
Added it to main! Thanks!